### ISSUE TITLE
about QVI model

### ISSUE THREAD

**Issue by XiaoyuShi97** [tarun005/FLAVR#21]
Hi. Thx for your efforts on benchmarking existing models. I wonder which repo you are using for quadratic video inpainting (QVI) model? Could you please share the link?

**Comment by tarun005** [comment]
https://www.dropbox.com/s/07kykjaw55pbwra/nips19_qvi_eval_release.zip?dl=0&file_subpath=%2Fnips19_qvi_eval_release

It is officially provided by the authors.

**Comment by XiaoyuShi97** [comment]
Thx for your prompt reply. I use this model and pretrained checkpoint on vimeo dataset, and got 31.85 for PSNR and 0.956 for ssim, which is lower than 35.15. Did I misunderstand anything?

**Comment by tarun005** [comment]
The preprocessing step involves normalizing with mean 0.5 and std 0.5, which can change the PSNR values. Other than that, it is exactly the same as the original code.

**Comment by tarun005** [comment]
Following the paper and code, we do a preprocessing normalizing with mean, std = (0.5,0.5). You have to apply the same for the model at test-time.

**Comment by XiaoyuShi97** [comment]
One more question. During training, t is fixed to be 0.5 and when testing, t are 1/8, 2/8, 3/8, 4/8, 5/8, 6/8, 7/8?

**Comment by tarun005** [comment]
For 2x interpolation, at train and test time t is fixed to be 0.5. For 8x interpolation, at train and test time, t is [1/8, 2/8, 3/8, 4/8, 5/8, 6/8, 7/8] - this is because models trained for 2x specifically do not transfer that well for multi frame interpolation ( they are strong baselines nevertheless). So our QVI models on GoPro and Vimeo are different.

**Comment by FR6024** [comment]
> Thx for your prompt reply. I use this model and pretrained checkpoint on vimeo dataset, and got 31.85 for PSNR and 0.956 for ssim, which is lower than 35.15. Did I misunderstand anything?

Hello, I have encountered the same problem. May I know how the pre processing area （normalizing with mean, std = (0.5,0.5) ）is specifically set in code?
